include/seqan3/utility/parallel/all.hpp

```diff
@@ -30,6 +32,7 @@
  *
  * \endif
```

Comment:
(Can't mark the correct lines)
Should the `\if DEV` block from line 28-33 be removed?

Reply:
I'd even say that just the whole file should just be `\\!cond DEV`ed?

Reply:
like so?

Reply:
if you put the cond above `/*!\file`, it shouldn't show up at all? 
https://seqan3-git-fork-marehr-devonly-seqan.vercel.app/usr/html/dir_8af16cb5769d23173486b3b2f0ac1c5d.html
it shows all the files.